code/__HELPERS/icons.dm

```diff
@@ -1273,3 +1273,11 @@ GLOBAL_LIST_EMPTY(transformation_animation_objects)
 	alert_overlay.transform = alert_overlay.transform.Scale(scale)
 
 	return alert_overlay
+
+/// Strips all lighting underlays from a mutable appearance.
+/proc/strip_light_underlays(mutable_appearance/appearance) as /mutable_appearance
+	RETURN_TYPE(/mutable_appearance)
```

Comment:
this does not take a mutable appearance, it takes any appearance. likewise, it does not return a mutable appearance, it returns an appearance. the mutable appearance cast is merely for safe var access